### PR TITLE
CNFT2-1605 Preview page: header

### PR DESCRIPTION
**Pull request by jlee-enquizit**
## Description

Preview page buttons and header tags for Published, Published with Draft, and Initial Draft page. This includes the introduction of the trussworks Tooltip. Button functionality forthcoming

## Tickets

* [CNFT2-1605](https://cdc-nbs.atlassian.net/browse/CNFT2-1605)
* [CNFT2-1937](https://cdc-nbs.atlassian.net/browse/CNFT2-1937)
* [CNFT2-1938](https://cdc-nbs.atlassian.net/browse/CNFT2-1938)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT2-1605]: https://cdc-nbs.atlassian.net/browse/CNFT2-1605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT2-1937]: https://cdc-nbs.atlassian.net/browse/CNFT2-1937?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT2-1938]: https://cdc-nbs.atlassian.net/browse/CNFT2-1938?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/page/management/preview/PreviewPage.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const PreviewPage = () => {
 
 const PreviewPageContent = () => {
     const { page, selected } = usePageManagement();
+    console.log(page.status);
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/management/preview/PreviewPage.tsx:31]: `Console.log`

File: apps/modernization-ui/src/components/button/link/LinkButton.tsx [hunk 2]
```diff
@@ -5,20 +5,22 @@ type Props = {
     href: string;
     label?: string;
     type?: 'outline' | 'solid';
+    unstyled?: boolean;
```
Comment (mpeels) [inline apps/modernization-ui/src/components/button/link/LinkButton.tsx:8]: Suggestion: Might be cleaner to just pass a `className` and change the styling that way.

**Review by mpeels — CHANGES_REQUESTED** [review]
See comments above

File: apps/modernization-ui/src/components/button/link/LinkButton.tsx (hunk 2 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/components/button/link/LinkButton.tsx:8]: There is that `type = 'outline` setting that's hard-coded in. I figured it cleanest to override with a prop but open to suggestions?

**Comment by jlee-enquizit** [comment]
image

**Review by mpeels — APPROVED** [review]

File: apps/modernization-ui/src/styles/_uswds.scss [hunk 3]
```diff
@@ -102,3 +102,22 @@ table.usa-table {
...
+            padding: 0.4rem 0.75rem;
+        }
+    }
+    // styling tooltip body
+    .usa-tooltip__body {
+        font-family: "Public Sans";
+        font-size: 1rem;
+        padding: 0.75rem;
+    }
+}
```
Comment (mpeels) [inline apps/modernization-ui/src/styles/_uswds.scss:123]: Issue: these are being applied globally. Anyone else in the app that uses the tooltip will have to fight with these style changes.

**Review by mpeels — CHANGES_REQUESTED** [review]

File: apps/modernization-ui/src/styles/_uswds.scss (hunk 3 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/styles/_uswds.scss:123]: this just strips out the most aggregious native styling by trussworks

**Review by mpeels — APPROVED** [review]

**Review by rkrusselenq — APPROVED** [review]

**Pull request merged**